src/main/java/org/traccar/protocol/MictrackLowAltitudeFlightProtocol.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 - 2018 Anton Tananaev (anton@traccar.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.protocol;
+
+import io.netty.handler.codec.string.StringDecoder;
+import io.netty.handler.codec.string.StringEncoder;
+import org.traccar.BaseProtocol;
+import org.traccar.PipelineBuilder;
+import org.traccar.TrackerServer;
+
+public class MictrackLowAltitudeFlightProtocol extends BaseProtocol {
```

Comment:
Ideally I would like to combine the decoder with existing `Mictrack`, so we don't have two ports for the same vendor.

Reply:
Well, I get your point, but to me the protocol format is very different from the Mictrack protocol. It has much more similarity with the Tlt2h protocol. If you still want to do it, would you be so kind to take the lead on that because I am not very familiar with the architecture of Traccar just yet.

Reply:
You just need to combine decoders.

Reply:
Do you know of an example where it is done how you would like to see it?

Reply:
You can take a look at H02 protocol. It supports two completely different formats. One binary and one text.

Reply:
Ok, I have merged the Mictrack Low Altitude Flight format into the existing Mictrack protocol decoder. Can you check if there is anything else that is missing? Thank you so much!